gamemodes/horde/gamemode/sv_economy.lua

```diff
@@ -228,6 +228,28 @@ function plymeta:Horde_DropMoney(amount)
     end
 end
 
+function plymeta:Horde_PayPlayer(plyToPay, amount)
+    if not plyToPay or not amount then return end
+    local plyToPay = tostring(plyToPay)
+    local amount = tonumber(amount)
+    local plyForMoney
+    local allPlys = player.GetAll()
+    for i=1, #allPlys do
+        local iterPly = allPlys[i]
+        local findString = string.find(string.lower(iterPly:GetName()), string.lower(plyToPay))
+        if findString then
+            plyForMoney = iterPly
+        end
+    end
+    if plyForMoney == nil then return end
+    if plyForMoney:SteamID() == self:SteamID() then return end -- Not that this really matters but it'd be a bit silly 
+    local amount = math.floor(amount)
+    if not amount or amount <= 0 or self:Horde_GetMoney() < amount then return end
+    self:Horde_AddMoney(-amount)
+    self:Horde_SyncEconomy()
+    plyForMoney:Horde_AddMoney(amount)
+    plyForMoney:Horde_SyncEconomy()
```

Comment:
Same for when you send money, some sort of chat confirmation would be nice, would also be nice for the receiver (with antispam in mind)